reference/7.0/Microsoft.PowerShell.Core/About/about_Windows_PowerShell_Compatibility.md

```diff
@@ -0,0 +1,94 @@
+---
+keywords: powershell,cmdlet
+locale: en-us
+ms.date: 1/10/2020
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_windows_powershell_compatibility?view=powershell-7&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about_Windows_PowerShell_Compatibility
+---
+# About Group Policy Settings
+
+## SHORT DESCRIPTION
+
+Describes the Windows PowerShell Compatibility funtionality for PowerShell
+
+## LONG DESCRIPTION
+
+Windows PowerShell Compatibility funtionality means that for modules in
+the `%windir%\system32\WindowsPowerShell\v1.0\Modules` folder,
+if the module manifest doesn’t indicate that module is compatible with
+PowerShell Core then that module will be loaded in a background
+Windows PowerShell 5.1 process and using implicit remoting reflected
+into current PowerShell session.
+
+Windows PowerShell Compatibility funtionality can be invoked in 3 ways:
+
+1. Explicitly using module import with `-UseWindowsPowerShell` parameter
+
+```powershell
+Import-Module -Name ScheduledTasks -UseWindowsPowerShell
+```
+
+1. Implicitly using module import by module name or path
+
+```powershell
+Import-Module -Name ServerManager
+```
```

Comment:
Indent this to line up under the 'I' in Implicitly

Reply:
fixed.